User strings corrections and improvements

radinamatic opened the pull request:
## Summary
Various relatively minor changes in user strings to consolidate, simplify and avoid tech jargon, mostly inspired by reviewing questions from translators on Crowdin in releases v0.13 and v0.14.

## References
Fixes https://github.com/learningequality/clearinghouse/issues/543

## Reviewer guidance
Check if changes make sense.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests


## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

radinamatic committed:
User strings tech debt

radinamatic committed:
Linter

indirectlylit commented on kolibri/core/assets/src/views/sync/SelectSyncSourceModal.vue:
```diff
@@ -85,7 +85,7 @@
       localNetworkDescription: {
         message:
-          'Sync facility data with another instance of Kolibri on your local network or the internet',
+          'Sync facility data with another Kolibri server in your local network or the internet',
```
I think "on a network" is more common than "in a network" https://ell.stackexchange.com/questions/40946/in-the-network-or-on-the-network

```suggestion
          'Sync facility data with another Kolibri server on your local network or the internet',
```

indirectlylit commented on kolibri/plugins/device/assets/src/views/ManageContentPage/SelectTransferSourceModal/SelectImportSourceModal.vue:
```diff
@@ -107,13 +107,13 @@
       networkDescription: {
         message:
-          'Import resources from another instance of Kolibri running on another device, either in the same local network or on the internet',
+          'Import resources from another Kolibri server running on a device in your local network, or on the internet',
```
```suggestion
          'Import resources from another Kolibri server running on a device on your local network or the internet',
```

indirectlylit commented on kolibri/plugins/device/assets/src/views/ManageContentPage/channelUpdateStrings.js:
```diff
@@ -6,7 +6,7 @@ export default createTranslator('ChannelUpdateStrings', {
   },
   notAvailableFromNetwork: {
-    message: 'This channel was not found on other instances of Kolibri',
+    message: 'This channel was not found on any Kolibri server in your network',
```
```suggestion
    message: 'This channel was not found on any Kolibri server on your network',
```

indirectlylit commented on kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetupMethod.vue:
```diff
@@ -124,19 +124,21 @@
         message: 'Full device',
         context:
-          'Create or import a device that will be a server to be used by coaches and learners',
+          'Device that will be fully featured Kolibri server used by admins, coaches and learners',
```
```suggestion
          'Device that will be a fully featured Kolibri server used by admins, coaches and learners',
```

indirectlylit commented on kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetupMethod.vue:
```diff
@@ -124,19 +124,21 @@
       fullDeviceTooltip: {
         message:
-          'Create or import a device that will be a server to be used by coaches and learners',
+          'Device will be fully featured Kolibri server used by admins, coaches and learners',
```
```suggestion
          'Device will be a fully featured Kolibri server used by admins, coaches and learners',
```

indirectlylit reviewed (commented):
in vs on

radinamatic committed:
Topics > Folders

radinamatic commented:
@indirectlylit Thank you for spotting the articles I managed to swallow (as usual)! 
I incorporated the rest of your feedback, and added the term swap we discussed (`Topic > Folder`).

indirectlylit reviewed (approved):
nice

radinamatic committed:
Finished > Completed

radinamatic commented on kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue:
```diff
@@ -273,7 +273,7 @@
       removeFromBookmarks: 'Remove from bookmarks',
       saveToBookmarks: 'Save to bookmarks',
-      markResourceAsFinished: 'Mark resource as finished',
+      markResourceAsFinished: 'Mark resource as completed',
```
Changed for consistency, thank you for flagging @dufcrule! 💯

jtamiace commented on kolibri/plugins/learn/assets/src/views/SearchBox.vue:
```diff
@@ -264,9 +264,9 @@
+        message: 'Folders',
         context:
-          'Learners can filter their searches for resources by type. In this case, topics.\n\nA topic is a collection of resources and other topics within a channel.',
+          'Learners can also search for comlplete folders.\n\nA folder is a collection of resources and other subfolders within a channel.',
```
"comlplete" >>> "complete"

radinamatic replied:
Spellchecker, spellchecker, spellchecker!!! 🤦🏽‍♀️ 
Thank you!

jtamiace reviewed (approved):
These are really great improvements, Radina!

radinamatic committed:
Typo
